fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -89,6 +89,17 @@ object ItemKicker {
     case FreeHtmlKickerWithLink(body, _) => Some(body)
     case _ => None
   }
+
+  /** Return the canonical display text for a kicker */
+  def kickerDisplayText(itemKicker: ItemKicker): Option[String] = itemKicker match {
```

Comment:
What is the original kickerText function used for elsewhere? I did a `grep kickerText **/*.scala` in `frontend` and it yielded no results.